File: app/src/main/java/com/zxcx/zhizhe/ui/home/attention/AttentionContract.java
package com.zxcx.zhizhe.ui.home.attention;

import com.zxcx.zhizhe.mvpBase.GetView;
import com.zxcx.zhizhe.mvpBase.IGetPresenter;
import com.zxcx.zhizhe.ui.card.hot.CardBean;

import java.util.List;

public interface AttentionContract {

    interface View extends GetView<List<CardBean>> {
    }

    interface Presenter extends IGetPresenter<List<CardBean>> {
    }
}

File: app/src/main/java/com/zxcx/zhizhe/ui/card/attention/AttentionContract.kt
package com.zxcx.zhizhe.ui.card.attention

import com.zxcx.zhizhe.mvpBase.INullGetPostPresenter
import com.zxcx.zhizhe.mvpBase.NullGetPostView
import com.zxcx.zhizhe.ui.card.hot.CardBean
import com.zxcx.zhizhe.ui.classify.ClassifyBean

interface AttentionContract {

    interface View : NullGetPostView<List<CardBean>>{
        fun getClassifySuccess(list: List<ClassifyBean>)
    }

    interface Presenter : INullGetPostPresenter<List<CardBean>>{
        fun getClassifySuccess(list: List<ClassifyBean>)
    }
}

